hardhat.config.js

```diff
@@ -41,6 +42,12 @@ module.exports = {
       accounts: getSigners(),
       allowUnlimitedContractSize: true,
     },
+    shadowfork: {
+      url: "http://35.241.160.82:8545",
```

Comment:
The URL should be moved to the config to we can delete the shawork completely IMO

Reply:
Done